### ISSUE TITLE
Add setting to control inline chat accessibility logic

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#39637]
We have added an accessible diff viewer for inline chat. Today, this is on when accessibility mode is on and off otherwise. We should have dedicated setting with on, off, and auto
